File: Fun_Problems/twoSum.java
import java.util.Arrays;

public class twoSum{
    public static void main(String[] args) {
    int[] nums = {2, 12, 11, 7, 15};
    int target = 9;
    System.out.print(Arrays.toString(two_sum(nums, target)));
    }

public static int[] two_sum(int[] nums, int target){
    int[] total = new int[2];
    
    for (int i = 0; i < nums.length; i++){
        int complement = target - nums[i];
        
        for (int j = 0; j < nums.length; j++){
            if (nums[j] == complement){
                total[0] = j;
                total[1] = i;
            }
        }
    }


    return total;

}
}

File: Fun_Problems/two_sum.py
def two_sum(nums, target):
    final = []  # Create an array to hold the indexes

    for index, value in enumerate(nums):
        complement = target - value

        if complement in nums:  # If the complemnt is found in nums then just add it to the array and end the loop
            final.append(index)
            final.append(nums.index(complement))
            return final
    

nums = [12, 7, 11, 2, 15]
target = 9
print(two_sum(nums, target))

# Or you can do this:

# def two_sum(nums, target):
#     final = {}

#     for index, value in enumerate(nums):
#         complement = target - value

#         if complement in nums:
#             final[value] = index
#             final[complement] = nums.index(complement)
#             return [final[value], final[complement]]
    
